lib/mix/tasks/phoenix_slime.gen.html.ex

```diff
@@ -0,0 +1,147 @@
+defmodule Mix.Tasks.PhoenixSlime.Gen.Html do
+  use Mix.Task
+
+  @shortdoc "Generates controller, model and views for an HTML based resource"
```

Comment:
mix phoenix_slime.gen.html # Generates controller, model and views for an HTML based resource using Slime templates

?